Marlin/Marlin_main.cpp

```diff
@@ -4236,9 +4244,9 @@ inline void gcode_G28() {
 
     setup_for_endstop_or_probe_move();
 
-    float measured_z = probe_pt(current_position[X_AXIS] + X_PROBE_OFFSET_FROM_EXTRUDER,
-                                current_position[Y_AXIS] + Y_PROBE_OFFSET_FROM_EXTRUDER,
-                                true, 1);
+    float measured_z = probe_pt(X_probe_location,
+                                Y_probe_location,
+                                stow, 1);
```

Comment:
Perhaps this should be `X_probe_location + X_PROBE_OFFSET_FROM_EXTRUDER` and `Y_probe_location + Y_PROBE_OFFSET_FROM_EXTRUDER`, applying to all coordinates, instead of adding them above. Because the `probe_pt` function always subtracts these offsets. If you make that change does the resulting probe seem sensible?